pkg/apis/mobilesecurityservice/v1alpha1/mobilesecurityserviceapp_types.go

```diff
@@ -0,0 +1,58 @@
+package v1alpha1
```

Comment:
This file can also be removed I think, as we're not installing an mss app in this PR, just the service and the db used by the service. I will update the PR to remove it.